components/questions.js

```diff
@@ -4,6 +4,7 @@ import RangeSlider from "react-bootstrap-range-slider";
 
 const DEFAULT_CONFIDENCE = 3;
 const DEFAULT_RESULT = {answer: null, comment: "", confidence: DEFAULT_CONFIDENCE};
+const MIN_LENGTH = 5;
```

Comment:
There is no way anyone makes a meaningful comment in five characters 🙂
Let's make it 50.

Reply:
yeah😂, I'll change it